changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** Changelog ***
 
+2021.05.10 - version 3.11.1
```

Comment:
Oops! I approved, and then I saw the date here. Also we could add the PR reference as the others. ;)

Reply:
Good catch, fixing in 0192283